src/cheerio_crawler.js

```diff
@@ -29,6 +29,7 @@ const DEFAULT_OPTIONS = {
             maxEventLoopOverloadedRatio: 0.7,
         },
     },
+    beforeRequestFunction: ({ request }) => request,
```

Comment:
The name `beforeRequestFunction` is not very descriptive - the word "request" in the name probably refers to the HTTP operation, but that's confusing since the function also accepts `Request` as parameter. How about calling this `prepareRequestFunction` ?

Reply:
Yes, I was referring to the HTTP operation. However, you are right that might confuse.